site/content/en/docs/manual/basics/creating_an_annotation_task.md

```diff
@@ -1,260 +1,252 @@
 
-### Start frame
+Added attributes will be available from the **Objects menu**:
 
-Frame from which video in task begins.
+![Attributes](/images/attributes.jpg)
 
-### Stop frame
+To add an attribute, do the following:
 
-Frame on which video in task ends.
+1. Go to the **Constructor** tab and click **Add attribute**.
 
-### Frame Step
+   ![Attributes](/images/attributes_01.png)
 
-Use this option to filter video frames.
-For example, enter `25` to leave every twenty fifth frame in the video or every twenty fifth image.
+2. In the **Name** field enter the name of the attribute.
+3. From the drop-down, select way to display the attribute in the **Objects menu**:
+   - `Radio` enables the selection of one option from several options.
+   - `Checkbox` enables the selection of multiple options.
+   - `Text` sets the attribute to a text field.
+   - `Number` sets the attribute to numerical field in the following format: `min;max;step`.
+4. In the **Attribute values** field, add attribute values. <br>To separate values use **Enter**.
+   <br>To delete value, use **Backspace** or click **x** next to the value name.
+5. (Optional) For mutable attributes, select **Mutable**.
 
-### Chunk size
+To delete an attribute, click **Delete attribute**.
 
-Defines a number of frames to be packed in a chunk when send from client to server.
-Server defines automatically if empty.
+### Select files
 
-Recommended values:
+There are several ways to upload files:
 
-- 1080p or less: 36
-- 2k or less: 8 - 16
-- 4k or less: 4 - 8
-- More: 1 - 4
+<!--lint disable maximum-line-length-->
 
-### Dataset Repository
+| Data source          | Description                                                                                                                                                                                                                                         |
+| -------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| My computer          | Use this option to select files from your laptop or PC. <br> To select file: <br>1. Click on the **Select files** field: <br>![Select files](/images/select_files.jpg). <br> 2. Select files to upload.                                             |
+| Connected file share | **Advanced option**. <br>Upload files from a local or cloud shared folder. <br>**Note**, that you need to mount a fileshare first. <br>For more information, see [Share path](/docs/administration/basics/installation/#share-path)                 |
+| Remote source        | **Advanced option**.<br>Enter a list of URLs (one per line) in the field. <br>If you want a `manifest.json` file, select the **Use cache**. <br> For more information, see [Dataset manifest.](/docs/manual/advanced/dataset_manifest/)             |
```

Comment:
Something wrong here. Could you please create a task with a list of URLs and see how it works?